operator/docs/lokistack/sop.md

```diff
@@ -308,3 +308,29 @@ The query queue is currently under high load.
 ### Steps
 
 - Increase the number of queriers
+
+## Lokistack Storage Schema Warning
+
+### Impact
+
+The LokiStack warns on a newer object storage schema being available for configuration.
+
+### Summary
+
+The schema configuration does not contain the most recent schema version and needs an update.
+
+### Severity
+
+`Warning`
+
+### Access Required
+
+- Console access to the cluster
+- Edit access to the deployed operator and Loki namespace:
+  - OpenShift
+    - `openshift-logging` (LokiStack)
+    - `openshift-operators-redhat` (Loki Operator)
```

Comment:
I think the user only needs access to the LokiStack resource and not the operator?

```suggestion
- Edit access to the namespace where the LokiStack is deployed:
  - OpenShift
    - `openshift-logging` (LokiStack)
```